mrjones2014/legendary.nvim
# feat: add support for item groups

Resolves: #223 

## How to Test

1. Try adding an item group under `config.keymaps`, `config.commands`, and `config.funcs`, make sure they work correctly
2. Try adding an item group under `config.itemgroups` and make sure they work correctly
3. Make sure that when filters are provided, they are passed onto the next select when an item group is selected

## Testing for Regressions

I have tested the following:
